ako-gateway-api/nodes/avi_model_l7_translator.go

```diff
@@ -31,6 +31,45 @@ import (
 	"github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pkg/utils"
 )
 
+func (o *AviObjectGraph) AddDefaultHTTPPS(key string) {
```

Comment:
can we rename the function to AddDefaultHTTPPolicySet for better readability

Reply:
Done